go/mysql/mysql56_gtid_set.go

```diff
@@ -656,5 +656,23 @@ func popInterval(dst *interval, s1, s2 *[]interval) bool {
 }
 
 func init() {
-	gtidSetParsers[Mysql56FlavorID] = parseMysql56GTIDSet
+	gtidSetParsers[Mysql56FlavorID] = func(s string) (GTIDSet, error) {
+		return parseMysql56GTIDSet(s)
+	}
+}
+
+// Subtract takes in two Mysql56GTIDSets as strings and subtracts the second from the first
+// The result is also a string.
+// An error is thrown if parsing is not possible for either GTIDSets
+func Subtract(lhs, rhs string) (string, error) {
+	lhsSet, err := parseMysql56GTIDSet(lhs)
+	if err != nil {
```

Comment:
This function was needed because we need to find the difference between the executed gtid set of the replica and primary to find the errant gtids. Earlier VTOrc used to run `select gtid_subtract` on the MySQL connection to the replica, but since we don't connect to the MySQL replica anymore, we have to use an internal function.

Reply:
There are sufficient tests that are added to this function to verify that it works just as exected.